### PR TITLE
Introduction to banded reports

### PR DESCRIPTION
**Pull request by MarySammal**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Introduction to Banded Reports
+owner: Mary Sammal
+---
+# Introduction to Banded Reports
+
+Banded reports provide a generalized report layout notion. When you preview a banded report, a report document is generated based on the report layout you designed and the data source you specified.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md]: "a generalized report layout notion" What does this mean?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+![eurd-win-report-bands-design-time-default-layout](../../../images/eurd-win-report-bands-design-time-default-layout.png)
+
+You can also add the following bands:
+
+- **Report Header** and **Report Footer**  
+
+    The **Report Header** is the report's first band (margins are "out-of-page" zones). Use this band to display the report's name, company logo, [date of creation, username](add-extra-information.md), etc.
+
+    The **Report Footer** finalizes the report's informative part. It is placed before the Page Footer and Bottom Margin on the report's last page. You can use the Report Footer band for report [summaries](shape-report-data\shape-data-expression-bindings\calculate-a-summary.md) or conclusions.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md]: "informative part" Explain.

Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md]: I realized that a Report Footer does not finalizes the report. The Page Footer finalizes. So, I'll change to 
The **Report Footer** is placed before the Page Footer and Bottom Margin on the report's last page. ...
Ok?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md [hunk 3]
```diff
@@ -4,15 +4,15 @@ owner: Mary Sammal
 ---
 # Introduction to Banded Reports
 
-Banded reports provide a generalized report layout notion. When you preview a banded report, a report document is generated based on the report layout you designed and the data source you specified.
+Banded reports provide a generalized report layout notion. A report document is generated based on the specified data source and the report layout you designed.
```
Comment (MarySammal) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md]: I cannot remove the "When you preview a banded report," part. At the time a reader reads this topic, he/she does not know that a report is what they design and a report document is generated based on the report layout they designed and the data source they specified, when they preview the report. If we remove this part, "A report document" that starts the next sentence is not clear for the reader.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md (hunk 2 above)
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md]: Ok

**Pull request merged**